modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -682,6 +682,11 @@ nagios_code=2
 
 case ${action} in
   push)
+    if [ "${dbms}" == "mongo" ] && [ "$(is_writable_mongo)" != "true" ]; then
+      trap - EXIT # remove the trap, as we do not want to send a nagios message in this case
+      log "This machine is not a mongo master. Skipping."
+      exit
+    fi
```

Comment:
Having a comment about why this is necessary might be useful

Reply:
I've got all the info in the commit message, and I wouldn't necessarily want to replicate `git blame` with super verbose comments in the code. Does that suffice?

Reply:
Having the motivation in the commit message is great. I guess the reasoning behind this is complicated enough that it's difficult to explain why this code is here concisely.

I'm fine with keeping things as they are.